GCDWebServer/Core/GCDWebServer.h

```diff
@@ -69,6 +69,12 @@ typedef GCDWebServerResponse* _Nullable (^GCDWebServerProcessBlock)(__kindof GCD
 typedef void (^GCDWebServerCompletionBlock)(GCDWebServerResponse* _Nullable response);
 typedef void (^GCDWebServerAsyncProcessBlock)(__kindof GCDWebServerRequest* request, GCDWebServerCompletionBlock completionBlock);
 
+/**
+ *  The GCDWebServerBuiltInLoggerBlock is used to override the built-in logger at runtime.
+ *  The message parameter is the formatted log message with the logging level.
+ */
+typedef void (^GCDWebServerBuiltInLoggerBlock)(NSString* _Nonnull message);
```

Comment:
I think the level should be passed as an argument separately from the message string to allow custom formatting.

Reply:
Should we pass the level name string or the actual level `GCDWebServerLoggingLevel`? Currently `GCDWebServerLoggingLevel` is a private type.